modules/nodes-pods-allocate-dra-configure-about.adoc

```diff
@@ -27,69 +27,150 @@ spec:
-    - name: 2g-10gb
-      deviceClassName: example-device-class
-      selectors:
-      - cel:
-          expression: "device.attributes['driver.example.com'].profile == '2g.10gb'"
-    - name: 3g-20gb
-      deviceClassName: example-device-class
-      selectors:
-      - cel:
-          expression: "device.attributes['driver.example.com'].profile == '3g.20gb'"
+    - name: req-0
+      exactly:
+      - name: 1g-5gb
+        deviceClassName: example-device-class
+        selectors:
+        - cel:
+            expression: "device.attributes['driver.example.com'].profile == '1g.5gb'"
+      - name: 2g-10gb
+        deviceClassName: example-device-class
+        selectors:
+        - cel:
+            expression: "device.attributes['driver.example.com'].profile == '2g.10gb'"
+      - name: 3g-20gb
+        deviceClassName: example-device-class
+        selectors:
+        - cel:
+            expression: "device.attributes['driver.example.com'].profile == '3g.20gb'"
+----
+
+Admin access::
+A cluster administrator can gain privileged access to a device that is in use by other users. This enables administrators to perform tasks such as monitoring the health and status of devices while ensuring that users can continue to use these devices with their workloads.
++
+To gain admin access, an administrator must create a resource claim or resource claim template with the `adminAccess: true` parameter in a namespace that includes the `resource.kubernetes.io/admin-access: "true"` label. Non-administrator users cannot access namespaces with this label. 
++
+.Example namespace with admin access label
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    resource.kubernetes.io/admin-access: "true"
+# ...
+----
++
+In the following example, the administrator is granted access to the `2g-10gb` device:
++
+.Example resource claim object with admin access
+[source,yaml]
+----
+apiVersion: resource.k8s.io/v1
+kind: ResourceClaimTemplate
+metadata:
+  name: large-black-cat-claim-template
+spec:
+  devices:
+    requests:
+    - name: req-0
+      exactly:
+        allocationMode: All
+        adminAccess: true
+        deviceClassName: example-device-class
+        selectors:
+        - cel:
+            expression: "device.attributes['driver.example.com'].profile == '2g.10gb'"
 ----
++
+--
+where:
+
+`spec.devices.requests.name.exactly.adminAccess:true`:: Specifies that the admin access mode is enabled for the specified device.
+--
+
+Priority lists::
+In both resource claim templates and resource claims, you can either require all requested devices be available before a pod can be scheduled, or create a prioritized list of two resources in case the first resource is not available. 
++
+In the `ResourceClaimTemplate` or `ResourceClaim` object, you must specify `exactly` to require all devices to be available or `firstAvailable` to create a priority list. 
```

Comment:
I think we should focus on the functionality of the options `exactly`, `firstAvailable` rather than insisting on the availability of the devices when these options are used.